File: JAVA/Phone.java
import java.util.*;

public class Phone {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter 1st three digit of your phone number");
        long y = sc.nextLong();
        System.out.println("Enter next three digit of your phonr number");
        long x = sc.nextLong();
        System.out.println("Enter last four digit of your phonr number");
        long n = sc.nextLong();
        long p = y * 40;
        long z = p * 25;
        long m = z + x;
        long a = m * 50;
        long c = a + 1;
        long d = c * 400;
        long w = d + n;
        long g = w + n;
        System.out.println(+g);
    }
}

File: C/phone.c
#include <stdio.h>
int main()
{
    printf("Enter 1st three digit of your phone number");
    // long y = in.nextLong();
    long int y;
    scanf("%ld", &y);
    printf("Enter next three digit of your phonr number");
    // long x = in.nextLong();
    long int x;
    scanf("%ld", &x);
    printf("Enter last four digit of your phonr number");
    // long  n = in.nextLong();
    long int n;
    scanf("%ld", &n);
    long int p = y * 40;
    long int z = p * 25;
    long int m = z + x;
    long int a = m * 50;
    long int c = a + 1;
    long int d = c * 400;
    long int w = d + n;
    long int g = w + n;
    printf("%ld", g);
    return 0;
}